medicines/doc-index-updater/Makefile

```diff
@@ -51,4 +55,4 @@ docker-push: ## Push Docker image
 
 .PHONY: help
 help: ## Display this help screen
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
\ No newline at end of file
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
wow

Reply:
this is great but it's not obvious it's here just from browsing the repo, so I agree a _To run tests_ section in the README would be great

Reply:
Done